ui/src/components/import/DataSheetView.js

```diff
@@ -392,6 +393,13 @@ const DataSheetView = ({jobId, onIngest}) => {
 
   const onGridReady = (p) => {
     setGridApi(p.api);
+
+    getResult('config/aggrid').then((result) => {
```

Comment:
It could be neater to return the ag-grid key along with the JWT on login. But then we'd need to persist it locally in a way that wasn't obvious.